tests/Backend/Feature/api/v1/OIDCTest.php

```
@@ -0,0 +1,2253 @@
+            ]),
+        ]);
+
+        // Simulate the state and nonce have been set in the session
+        Session::put('openid_connect_state', $state);
+        Session::put('openid_connect_nonce', $nonce);
+
+        $header = [
+            'Accept-Language' => 'fr',
+        ];
+
+        $response = $this->get(route('auth.oidc.callback', [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$response'. (Unused Code Rules)

(UnusedLocalVariable)